packages/cli/test/integration/commands/import.cmd.test.ts

```diff
@@ -1,16 +1,18 @@
-import * as testDb from '../shared/testDb';
-import { mockInstance } from '../shared/utils/';
-import { InternalHooks } from '@/InternalHooks';
-import { ImportWorkflowsCommand } from '@/commands/import/workflow';
 import * as Config from '@oclif/config';
+import { mock } from 'jest-mock-extended';
+import { type ILogger, LoggerProxy } from 'n8n-workflow';
 
-import { LoggerProxy } from 'n8n-workflow';
-import { getLogger } from '@/Logger';
+import { InternalHooks } from '@/InternalHooks';
+import { ImportWorkflowsCommand } from '@/commands/import/workflow';
+import { LoadNodesAndCredentials } from '@/LoadNodesAndCredentials';
+import * as testDb from '../shared/testDb';
+import { mockInstance } from '../shared/utils/';
 
-LoggerProxy.init(getLogger());
+LoggerProxy.init(mock<ILogger>());
```

Comment:
I love this, we should be mocking the logger more often.

Reply:
[I have a branch to convert Logger into a service](https://github.com/n8n-io/n8n/compare/refactor-logger?expand=1), and mock that everywhere. no more `LoggerProxy` in the `cli` package.